support-frontend/assets/components/subscriptionCheckouts/address/postcodeFinder.jsx

```diff
@@ -76,21 +76,22 @@ class PostcodeFinder extends Component<PropTypes> {
     const {
       id, postcode, results, isLoading, setPostcode, fetchResults, error, onPostcodeUpdate, onAddressUpdate,
     } = this.props;
-    return (
-      <div>
-        <ComposedInputWithButton
-          error={error}
-          label="Postcode"
-          onClick={() => { fetchResults(postcode); }}
-          id={id}
-          setValue={(val) => {
+    return true ? '' :
```

Comment:
I probably wouldn't put a switch inside the component itself, but in the place where the component sits. I would make the page responsible for determining whether to render it. But then I guess you have to implement it three times, which is more tedious to manage, so: okay.

Is there any way to change this so that it not a raw boolean that might be confusing to future devs?